src/devices/legrand.js

```diff
@@ -46,11 +46,10 @@ const fzlocal = {
 
 module.exports = [
     {
-        zigbeeModel: [' Pocket remote\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'+
-            '\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'],
+        fingerprint: [{modelID: ' Wireless Scenes Command\u0000\u0000\u0000\u0000\u0000\u0000\u0000'}],
```

Comment:
This removes support for the `Pocket remote`, please add `Wireless Scenes Command` to it.

Reply:
Sorry, I understand that this remove the support for Pocket Remote, but I don't get how to add Wireless Scenes Command to it. Could you please explain me ?

Reply:
Can you confirm your device also has model `067755` written on it?

Reply:
Yes, it's written on a sticker on it

Reply:
Thanks for the modification.